core/src/main/java/org/kiwix/kiwixmobile/core/data/local/entity/NetworkLanguageSpec.kt

```diff
@@ -15,14 +15,13 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  *
  */
+package org.kiwix.kiwixmobile.core.data.local.entity
 
-package org.kiwix.kiwixmobile.core.data.local.entity;
-
-import com.yahoo.squidb.annotations.TableModelSpec;
+import com.yahoo.squidb.annotations.TableModelSpec
 
 @TableModelSpec(className = "NetworkLanguageDatabaseEntity", tableName = "networkLanguages")
-public class NetworkLanguageSpec {
-  public String languageISO3;
-  public boolean enabled;
-  public int numberOfOccurences;
+class NetworkLanguageSpec {
+  var languageISO3: String? = null
+  var enabled = false
```

Comment:
```
var enabled = false
```
@kelson42 Should we by default set if `Language` is active to false?

Reply:
@gouri-panda I have no clue what your question implies. I don't know what this module is for. Please explain.

Reply:
This `enabled` allows us to check if the language is selected or not. This helps us to download the selected language's zim files. I looked at the code we are adding local languages too in the language list. So by default, we should turn it off.